### PR TITLE
remove repetitive code

### PR DESCRIPTION
**Pull request by geekvest**

### CODE REVIEW

File: crates/bevy_gizmos/src/lib.rs [hunk 1]
```diff
@@ -14,7 +14,7 @@
 //!
 //! See the documentation on [Gizmos](crate::gizmos::Gizmos) for more examples.
 
-/// Label for the the render systems handling the
+/// Label for the render systems handling the
```
Comment (TrialDragon) [inline crates/bevy_gizmos/src/lib.rs:17]: It's potentially unrelated, but I think this comment is incomplete and doesn't say all that it should. Or there is just a random redundant `the` at the end of it. Or both.

**Review by TrialDragon — APPROVED** [review]
LGTM for the most part. I had one question, but it's more than likely unrelated to this PR and is its own issue.

File: crates/bevy_gizmos/src/lib.rs (hunk 1 above)
Comment (geekvest) [reply crates/bevy_gizmos/src/lib.rs:17]: Thanks for the review. Should I revert the changes here?

Comment (james7132) [reply crates/bevy_gizmos/src/lib.rs:17]:
```suggestion
/// System set label for the systems handling the rendering of gizmos.
```

**Review by james7132 — APPROVED** [review]

**Comment by james7132** [comment]
Can you please either clear out or fill out the PR description? The full text gets added to the commit when merged.

**Review by afonsolage — APPROVED** [review]

**Review by mockersf — APPROVED** [review]

**Comment by geekvest** [comment]
> Can you please either clear out or fill out the PR description? The full text gets added to the commit when merged.
> 
> Also welcome to contributing to Bevy!

Thanks! Please review again.

**Review by james7132 — APPROVED** [review]

**Pull request merged**